src/database/database.hpp

```diff
@@ -29,7 +30,7 @@ class Database : public aos::iam::certhandler::StorageItf {
      * @param dbPath path to the database file.
```

Comment:
Please add doxygen for the new param
```suggestion
     * @param migrationPath path to the migration scripts.
```

Reply:
Done